packages/faceted-search/src/components/ClearBasicSearch/ClearBasicSearch.scss

```diff
@@ -0,0 +1,10 @@
+.tc-faceted-clear-basic {
+	border-right: 0.1rem solid $gallery;
```

Comment:
The button is placed at the end of the flex row. Why do we have a border on the right ?

Reply:
Add the screenshot on the JIRA